tests/commands/dev/test_run_dev_app.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from subprocess import CalledProcessError
+import pytest
+
+from briefcase.exceptions import BriefcaseCommandError
+
+
+def raise_called_process_error(*args, **kwargs):
+    raise CalledProcessError(returncode=2, cmd="cmd")
+
+
+def test_subprocess_running_successfully(dev_command, first_app):
+    env = dict(a=1, b=2, c=3)
+    dev_command.run_dev_app(first_app, env)
+    dev_command.subprocess.run.assert_called_once_with([sys.executable, '-m', first_app.app_name],
+                                                       env=env,
+                                                       check=True)
```

Comment:
We are aiming to follow "black" style indentation, not pycharm "hanging" indents. The eventual goal of actually applying black consistently across the codebase.